.travis.yml

```diff
@@ -1,13 +1,15 @@
 language: php
 php:
   - '7.1'
+  - '7.2'
+  - '7.3'
 sudo: required
 dist: trusty
 
 branches:
```

Comment:
We can remove branches restriction since we are creating a new branch per release.